packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -42,7 +42,7 @@ export const DeleteFilesControls = (props: {
     data: {
       taskCounts,
       tableData,
-      isActionStartDisabled: disablePrimary,
+      isActionStartDisabled: disableStart,
```

Comment:
Probably a follow up pr, but we shouldalign the useView hooks "disableStart"  with the controls context version "isActionStartDisabled"